rotkehlchen/exchanges/data_structures.py

```diff
@@ -442,6 +442,19 @@ def identifier(self) -> str:
         )
         return hash_id(string)
 
+    def serialize(self) -> Dict[str, Any]:
```

Comment:
why do you need this?

Reply:
Because when exporting the events, not all events returned are serializable. 
Even though they're subclasses of `AccountingEventMixin`

Reply:
So you are exporting `AccountingEventMixin` and not the end result of processing so the ProcessedHistory events? That's good.

But in that case if `serialize()`should be a method of `AccountingEventMixin` you should also add it to its interface in `accounting/mixins/event.py`